rosbag2_py/src/rosbag2_py/_rosbag2_py.cpp

```diff
@@ -0,0 +1,256 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
```

Comment:
Can we easily split this into two files? I could see that these classes (`Reader` and `Writer`) grow over time and I think it makes sense to have them separated into its own file respectively.

Reply:
54f6722 splits into `_reader`, `_writer`, and `_storage` files. Let me know if these are the final naming we want for the files and pybind targets. Storage includes more than just `rosbag2_storage`, so maybe it could be called utilities.